cmd/bootstrap/cmd/partner_infos.go

```diff
@@ -20,43 +18,40 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-var (
-	flagOutputDir    string
-	flagANAddress    string
-	flagANNetworkKey string
-	flagNetworkEnv   string
-
-	getNodeInfoScript = []byte(`import FlowIDTableStaking from 0x9eca2b38b18b5dfe
-
-	// This script gets all the info about a node and returns it
-	
-	pub fun main(nodeID: String): FlowIDTableStaking.NodeInfo {
-		return FlowIDTableStaking.NodeInfo(nodeID: nodeID)
-	}`)
-)
-
 const (
-	// Index of each field in the cadence NodeInfo as it corresponds to cadence.Struct.Fields needed to build NodePubInfo struct,
+	// Index of each field in the cadence NodeInfo as it corresponds to cadence.Struct.Fields
 	// fields not needed are left out.
 	idField = iota
 	roleField
 	networkingAddressField
 	networkingKeyField
 	stakingKeyField
-	tokensStakedField
+)
+
+const (
 	flowNodeAddrPart    = "nodes.onflow.org"
 	defaultPartnerStake = 100
 )
 
-// NodePubInfo basic representation of node-pub-info.json data
-type NodePubInfo struct {
-	Role          string `json:"Role"`
-	Address       string `json:"Address"`
-	NodeID        string `json:"NodeID"`
-	Stake         string `json:"Stake"`
-	NetworkPubKey string `json:"NetworkPubKey"`
-	StakingPubKey string `json:"StakingPubKey"`
-}
+var (
+	flagANAddress    string
+	flagANNetworkKey string
+	flagNetworkEnv   string
+
+	getInfoForProposedNodesScript = []byte(`
+		import FlowIDTableStaking from 0x9eca2b38b18b5dfe
```

Comment:
If we hard-code the address here, this will only work on Mainnet. We should set this address based on the `network` flag

Reply:
Yea you are too quick lol was just fixing that https://github.com/onflow/flow-go/pull/1988/commits/3d92335a62550b9005aee7dcb634b842450c02e5